packages/jaeger-ui/src/components/SearchTracePage/SearchForm.test.js

```diff
@@ -139,6 +147,7 @@ describe('submitForm()', () => {
       resultsLimit: 20,
       service: 'svc-a',
     };
+    trackEvent.mockClear();
```

Comment:
I'd say this might be better if it's in the test case... To make it clear it is only relevant in one test case?